System/Posix/Directory/Common.hsc

```diff
@@ -22,6 +22,9 @@
 
 module System.Posix.Directory.Common (
        DirStream(..), CDir, CDirent, DirStreamOffset(..),
+#ifdef HAVE_FDOPENDIR
+       openDirStreamFd,
```

Comment:
IMO, this should be in its own module `System.Posix.Directory.Fd`, no?

Reply:
Done